tests/antithesis/test-template/Dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG GO_VERSION=1.24.2
+ARG ARCH=amd64
+
+FROM golang:$GO_VERSION AS build
+
+# Copy replace directives for build
+# TODO - there should be a way to automate this
+ARG BUILD_DIR=/building
+
+WORKDIR $BUILD_DIR
+COPY . .
+
+# Run go mod tidy to ensure the dependencies are correct
+RUN go mod tidy
+
+# Building entrypoint 
+RUN cd $BUILD_DIR/tests && go build -o /opt/entrypoint -race ./antithesis/test-template/entrypoint/entrypoint.go
+
+# Building serial_driver_delete_keys
```

Comment:
Not a huge fun of comments that just repeat operation. What information `$ Building serial_driver_delete_keys` provides that `go build -o /opt/serial_driver_delete_keys -race ./antithesis/test-template/go-delete-keys/serial_driver_delete_keys.go` doesn't?

Reply:
Note; this applies to all comments in this dockerfile.

Reply:
Oh, didn't notice that this was present in previous dockerfile. Feel free to ignore this comment for this PR. We can address it in next iteration.